requirements.txt

```diff
@@ -1,2 +1,3 @@
 nomad-lab>=1.2.2.dev
 pytest
+rdkit
```

Comment:
I'm not sure how troublesome `rdkit` installation is in different platforms (windows, macos, linux). I guess we will see if it gives us trouble in the prod deployments.